app/assets/src/components/views/samples/ReportsDownloader.js

```diff
@@ -4,8 +4,14 @@ import { openUrl } from "~utils/links";
 import Cookies from "js-cookie";
 
 export default class ReportsDownloader {
-  constructor({ projectId = "all", onDownloadFail = null, downloadOption }) {
+  constructor({
+    projectId = "all",
+    onDownloadFail = null,
+    downloadOption,
+    selectedSampleIds
+  }) {
     this.projectId = projectId;
+    this.selectedSampleIds = Array.from(selectedSampleIds);
```

Comment:
Should we impose a limit on the number of samples? 
The url may become too large.

Reply:
Good question. Do we currently have a limit on the sample selection for the heatmap? If not, I'd suggest punting on the limit for now

Reply:
My concern is that it can crash. But apparently, the limits are not as strict as they used to be it, for chrome in particular...
So lets leave as is for now.

Reply:
Ok sounds good. Let's see how it goes